pom.xml

```diff
@@ -85,6 +85,19 @@
       <scope>test</scope>
       <type>jar</type>
     </dependency>
+    <dependency>
+      <groupId>org.apache.axis</groupId>
+      <artifactId>axis</artifactId>
+      <version>1.4</version>
+      <type>jar</type>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.13-SNAPSHOT</version>
```

Comment:
NIT: using a SNAPSHOT version of junit? Any specific purpose? It might trigger issues later with build reproducibility is that one gets wiped out (as SNAPSHOTs do).